src/libraries/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEntry.Unix.cs

```diff
@@ -45,15 +45,17 @@ namespace System.IO.Enumeration
             if (isUnknown)
             {
                 isSymlink = entry.IsSymbolicLink;
-                isDirectory = entry._status.IsDirectory(entry.FullPath);
+                // Need to fail silently in case we are enumerating
```

Comment:
I don't understand the comment... isn't all of this code about enumerating?

Reply:
Enumeration always goes through this path, yes. I do not know if in the future we will reach `FileSystemEntry.Initialize` another way.

Next time I touch this code, I can change the comment to `// This call needs to fail silently`.